__tests__/app.test.js

```diff
@@ -141,3 +141,57 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("should return status 201 and the new comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "butter_bridge",
+        body: "The most amazing comment ever",
+      })
+      .expect(201)
+      .then(({ body: { newComment } }) => {
+        expect(newComment).toMatchObject({
+          author: "butter_bridge",
+          body: "The most amazing comment ever",
+        });
```

Comment:
This is a good start, but we should be expecting more than just these two properties. When this value is inserted into the database what other properties should we expect it to have?